Modules/Loadable/Segmentations/MRMLDM/vtkMRMLSegmentationsDisplayableManager2D.cxx

```diff
@@ -781,11 +793,37 @@ void vtkMRMLSegmentationsDisplayableManager2D::vtkInternal::UpdateDisplayNodePip
       pipeline->PolyDataOutlineActor->SetVisibility(false);
       pipeline->PolyDataFillActor->SetVisibility(false);
 
+      // Set the range of the scalars in the image data from the ScalarRange field if it exists
+      // Default to the scalar range of 0.0 to 1.0 otherwise
+      double minimumValue = 0.0;
+      double maximumValue = 0.0;
+      vtkSmartPointer<vtkDoubleArray> scalarRange = vtkDoubleArray::SafeDownCast(
```

Comment:
You can use a simple pointer here, as the data set owns the array and it won't release it while we are reading the values from it.